src/GovUk.Education.ExploreEducationStatistics.Data.Api/Startup.cs

```diff
@@ -179,9 +181,12 @@ public void ConfigureServices(IServiceCollection services)
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         {
-            // Enable caching and register any caching services
+            // Enable caching and register any caching services.
             CacheAspect.Enabled = true;
             BlobCacheAttribute.AddService("default", app.ApplicationServices.GetService<IBlobCacheService>());
+            // Enable cancellation aspects and register request timeout configuration.
+            AddTimeoutCancellationAspect.Enabled = true;
+            AddTimeoutCancellationAttribute.SetTimeoutConfiguration(Configuration.GetSection("RequestTimeouts"));
```

Comment:
I'm only enabling the aspect here because that allows us to add the timeout behaviour to the public APIs, whereas the Admin doesn't use the timeouts (yet).

The filter is used to handle the user-initiated cancellations.  Although in theory the user should never see these responses, it does prevent a big stacktrace getting logged out!

Reply:
Got it, thanks. I think it's worth dropping a note to @luke-h1 to mention that the timeout feature won't apply to the Admin just yet, for when it comes to testing.

Reply:
Will do, ta